common/evolving.cpp

```diff
@@ -0,0 +1,321 @@
+/*	EQEMu: Everquest Server Emulator
+	Copyright (C) 2001-2003 EQEMu Development Team (http://eqemulator.net)
+
+	This program is free software; you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation; version 2 of the License.
+
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY except by those people which sell it, which
+	are required to give you total support for your newly bought product;
+	without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+	A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with this program; if not, write to the Free Software
+	Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+*/
+
+#include "evolving.h"
+#include "item_instance.h"
+#include "events/player_event_logs.h"
+#include "repositories/character_evolving_items_repository.h"
+
+EvolvingItemsManager::EvolvingItemsManager()
+{
+	m_db         = nullptr;
+	m_content_db = nullptr;
+}
+
+void EvolvingItemsManager::LoadEvolvingItems() const
+{
+	auto const &results = ItemsEvolvingDetailsRepository::All(*m_db);
```

Comment:
Maybe.   I made the assumption that it would move to a content table and be static based on in era items.  If that is the decision taken, and we get a content table made and populated, I will update and re-test accordingly.  For my testing I used a local table.